doc/sphinxman/source/dft.rst

```diff
@@ -531,16 +530,18 @@ octohedral, and icosohedral systems are not complete, so there may be some
 ambiguity in the grid orientation for these systems.
 
 Radial grid types are controlled by the |scf__dft_radial_scheme| option, which
-at the moment may be either TREUTLER or BECKE, while the number of radial points
-are controlled by the |scf__dft_radial_points| option, which is any positive
+at the moment may be either ``TREUTLER`` or ``BECKE``, while the number of radial points are controlled by the |scf__dft_radial_points| option, which is any positive
 integer (typically 50-100). The radial grids are "centered" on the Bragg-Slater
 radius of each atom, as described in Becke's 1988 paper. If inaccurate
 integration is suspected in systems with anions or very diffuse basis functions,
-the |scf__dft_bs_radius_alpha| option may be increased from 1.0 to a larger value to
-force the radial grid to span a larger extent in space.
+the |scf__dft_bs_radius_alpha| option may be increased from 1.0 to a larger value to force the radial grid to span a larger extent in space.
 
 The atomic weighting scheme is controlled by the |scf__dft_nuclear_scheme|
-option, which may be one of TREUTLER, BECKE, or NAIVE.
+option, which may be one of ``TREUTLER``, ``BECKE``, ``STRATMANN``, ``NAIVE``, or ``SBECKE``. The latter is a smoother variant of the BECKE scheme recently introduced by Laqua [Laqua:2018:204111]_ that should behave better for weak interactions.
+
+Pruning of the quadrature grid is controlled by the |scf__dft_pruning_scheme|
+option. The options ``ROBUST`` and ``TREUTLER`` devide the grid into spherical regions
+based on the Bragg-Slater radius of each atom and apply different orders to them. The ``ROBUST`` scheme is a less aggressive variant of the ``TREUTLER`` approach and suitable for benchmark-level quality (MAD < 0.002 kcal/mol for the S22 with PBE/aug-cc-pVTZ for pruned versus unpruned grid). Our implementation of the ``TREUTLER`` scheme shows an error of 0.02 kcal/mol for the same bechmark. Both also reduce the grid order by 1 for H and He atoms. Other schemes mentioned in the keyword documentation (e.g. P_SLATER) are experimental and should be considered expert-only.
```

Comment:
This needs newlines like above.

We should add a note here that the pruning option was introduced in `v1.4` and may change the energies compared to previous versions by the aforementioned 0.002 kcal/mol relative or 1.e-6 absolute.

@cdsgroup we may need to recruit someone to do a deeper investigation here.

Reply:
added a note on top.